packages/aws-cdk/test/api/logs/cli-logging.test.ts

```diff
@@ -1,10 +1,16 @@
-import { LogLevel, log, setLogLevel, setCI, data, print, error, warning, success, debug, trace, prefix, withCorkedLogging } from '../lib/logging';
+import { LogLevel, log, setLogLevel, setCI, data, print, error, warning, success, debug, trace, prefix, withCorkedLogging } from '../../../lib/logging';
 
 describe('logging', () => {
   // Mock streams to capture output
   let mockStdout: jest.Mock;
   let mockStderr: jest.Mock;
 
+  // Helper function to strip ANSI codes
+  const stripAnsi = (str: string): string => {
```

Comment:
I have this function twice, I figure it's fine but if it's not DRY enough then I can put it in a helper function directory

Reply:
I don't mind.